src/ebay-menu/menu.tsx

```diff
@@ -1,90 +1,145 @@
 import React, {
-    Children, cloneElement, useEffect, useState,
-    FC, ReactElement
+    Children,
+    cloneElement,
+    FC,
+    KeyboardEvent,
+    MouseEvent,
+    ReactElement,
+    useEffect,
+    useState
 } from 'react'
 import classNames from 'classnames'
 import useRovingIndex from '../common/event-utils/use-roving-index'
-import { usePrevious } from '../common/component-utils/usePrevious'
-import { handleActionKeydown } from '../common/event-utils'
-import { MenuItemProps } from './menu-item'
-import { EbayMenuItem } from './index'
-import { MenuProps } from './index'
+import { isActionKey } from '../common/event-utils'
+import { withForwardRef } from '../common/component-utils'
+import { EbayMenuItem, MenuItemProps, EbayMenuProps } from './index'
+import { Key } from '../common/event-utils/types'
 
-
-const changedIndex = (arr1: boolean[], arr2: boolean[]): number => arr1.findIndex((x, i) => arr2[i] !== x)
-
-const EbayMenu: FC<MenuProps> = ({
-    baseEl: Container = 'span',
```

Comment:
Maybe this got removed because of the different branch? We added in this PR https://github.com/eBay/ebayui-core-react/pull/231

Reply:
merge mistake, fixed back